mixpanel.go

```diff
@@ -124,7 +128,37 @@ func (m *mixpanel) Track(distinctId, eventName string, e *Event) error {
 	return m.send("track", params, autoGeolocate)
 }
 
-// Updates a user in mixpanel. See
+// Import create an event for an existing distinct id, even if the event is older than 5 days.
+// See https://developer.mixpanel.com/docs/importing-old-events
+func (m *mixpanel) Import(distinctId, eventName string, e *Event) error {
+	if e.Timestamp == nil || time.Since(*e.Timestamp) < 5*24*time.Hour {
```

Comment:
This feels opinionated for a client library – will Mixpanel reject the events in this case? Otherwise I'd say it's not up to the library to police

Reply:
Maybe there is a wrapper function called ImportOrTrack that contains this logic?